packages/react/src/components/ExpandCollapsePanel/index.css

```
@@ -1,5 +1,35 @@
+:root {
+  --expandcollapse-trigger-background-color: var(--gray-10);
+  --expandcollapse-trigger-color: var(--gray-90);
+  --expandcollapse-panel-background-color: var(--gray-10);
+  --expandcollapse-panel-color: var(--gray-90);
```

```suggestion
  --expandcollapse-trigger-background-color: transparent;
  --expandcollapse-trigger-color: var(--gray-90);
  --expandcollapse-panel-background-color: transparent;
  --expandcollapse-panel-color: var(--gray-90);
```

This component isn't intended to be styled by default. The only thing we really need to set is the appropriate text color for light/dark backgrounds.